modules/cnv-creating-vddk-image.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-importing-vmware-vm.adoc
+
+[id="cnv-creating-vddk-image_{context}"]
+= Creating the Virtual Disk Development Kit image
+
+You can download the VMware Virtual Disk Development Kit (VDDK) SDK, convert it into an image, and push it to a container repository.
+
+.Prerequisites
+
+* *My VMware* credentials to download the VDDK SDK
+* Private container repository. Storing the VDDK SDK in a public repository may violate the terms of the VMware license.
```

Comment:
s/Private container repository./You need a private container repository./

Reply:
Same as above.